src/utils.jl

```diff
@@ -295,10 +295,11 @@ function transfer_solution(plotter::MakiePlotter{3}, u::Vector; field_idx::Int=1
 
     current_vertex_index = 1
     data = fill(0.0, num_vertices(plotter), field_dim)
-    for (cell_index, cell) in enumerate(Ferrite.getcells(plotter.dh.grid))
+    for cell in Ferrite.CellIterator(dh)
+        cell_index = cell.current_cellid.x
```

Comment:
Out of curiosity, why is this changed?

Reply:
now its consistent :D

Reply:
but I can also change it back and change instead the 2d version to also use the iteration over the `cells` array

Reply:
actually, lets change to the 2D-like version without `CellIterator` because the version without it is `AbstractGrid` agnostic